<ui version="4.0" >
 <class>Mn2NcDialog</class>
 <widget class="QWidget" name="Mn2NcDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Meteonorm - netCDF Converter</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="mnFileFrame" >
     <property name="title" >
      <string>Meteonorm file</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="mnFileNameLabel" >
          <property name="text" >
           <string>File name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="mnFileNameInput" />
        </item>
        <item>
         <widget class="QToolButton" name="mnFileNameDialogButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="mnFileLoadButton" >
          <property name="text" >
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpButton" >
          <property name="text" >
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="locationFrame" >
     <property name="title" >
      <string>Location</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="4" column="2" colspan="2" >
       <widget class="QPushButton" name="earthViewButton" >
        <property name="text" >
         <string>Show 3D view</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QSpinBox" name="locHeightInput" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> m</string>
        </property>
        <property name="maximum" >
         <number>10000</number>
        </property>
        <property name="minimum" >
         <number>-100</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="locLatitudeInput" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> °</string>
        </property>
        <property name="maximum" >
         <double>90.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-90.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="2" >
       <widget class="QLabel" name="locTimeZoneLabel" >
        <property name="text" >
         <string>Time zone:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="locHeightLabel" >
        <property name="text" >
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="locLatitudeLabel" >
        <property name="text" >
         <string>Latitude:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QSpinBox" name="locTimeZoneInput" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> h</string>
        </property>
        <property name="prefix" >
         <string/>
        </property>
        <property name="maximum" >
         <number>23</number>
        </property>
        <property name="minimum" >
         <number>-23</number>
        </property>
        <property name="singleStep" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="locNameLabel" >
        <property name="text" >
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="locCommentLabel" >
        <property name="text" >
         <string>Comment:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3" >
       <widget class="QLineEdit" name="locCommentInput" />
      </item>
      <item row="0" column="1" colspan="3" >
       <widget class="QLineEdit" name="locNameInput" />
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="locLongitudeLabel" >
        <property name="text" >
         <string>Longitude:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QDoubleSpinBox" name="locLongitudeInput" >
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="suffix" >
         <string> °</string>
        </property>
        <property name="maximum" >
         <double>180.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-180.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statisticsFrame" >
     <property name="title" >
      <string>Statistics</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="statisticsButton" >
        <property name="text" >
         <string>Enable statistical calculations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="statisticsBrowser" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ncFileFrame" >
     <property name="title" >
      <string>netCDF file</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ncFileNameLabel" >
          <property name="text" >
           <string>File name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ncFileNameInput" />
        </item>
        <item>
         <widget class="QToolButton" name="ncFileNameDialogButton" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="ncFileSaveButton" >
          <property name="text" >
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>statisticsButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>statisticsBrowser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>66</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel" >
     <x>86</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
